setup.py

```diff
@@ -1,14 +1,25 @@
 from setuptools import setup, find_packages
+import re
```

Comment:
from ena_upload._version import __version__

does not work here?

Reply:
@bgruening it does, way better, thanks!